### DISCUSSION TITLE
Glitched rendering of editor

### DISCUSSION THREAD

**Discussion by Kuhaku123456** [microsoft/vscode-discussions#1772] [Extension Development QnA]
![Screenshot 2024-10-01 075158](https://github.com/user-attachments/assets/23c36cdb-64c4-48ae-96fd-ad6bd075df7b)

What happened with my VS Code
only happens when opening vs code no problem when opening other applications

**Comment by robconery** [comment]
This looks to be a localized rendering issue that could result from a number of problems, including graphics card problems (I'm thinking drivers). Here's a Reddit thread that might help: https://www.reddit.com/r/vscode/comments/szrn3m/weird_visual_glitch_on_vs_code_and_other_similar/

**Comment by gjsjohnmurray** [comment]
Please also see https://github.com/microsoft/vscode-discussions/discussions/5 to learn about the specific focus of this forum and to find sources of general VS Code support.
